modules/blocks.php

```diff
@@ -190,3 +191,70 @@ function jetpack_slideshow_block_load_assets( $attr, $content ) {
 	Jetpack_Gutenberg::load_assets_as_required( 'slideshow', $dependencies );
 	return $content;
 }
+
+/**
+ * Business Hours Block.
+ */
+jetpack_register_block(
+	'business-hours',
+	array( 'render_callback' => 'jetpack_business_hours_render' )
+);
+/**
+ * Business Hours Block dynamic rending of the glock.
+ *
+ * @param array  $attr - Array containing the business hours block attributes.
+ * @param string $content - String containing the business hours block content.
+ *
+ * @return string
+ */
+function jetpack_business_hours_render( $attributes, $content ) {
+	global $wp_locale;
+
+	if ( empty( $attributes['hours'] ) || ! is_array( $attributes['hours'] ) ) {
+		return $content;
+	}
+
+	$start_of_week = (int) get_option( 'start_of_week', 0 );
+	$time_format = get_option( 'time_format' );
+	$today = current_time( 'D' );
+	$content = '<dl class="business-hours built-by-php">';
```

Comment:
Could you add `$attr['className']` if it is not empty here? It allows one to use the field provided by the block editor here:

![image](https://user-images.githubusercontent.com/426388/52214485-02dec180-2892-11e9-8858-cbd650646e0c.png)